doc/release-notes.md

```diff
@@ -1,5 +1,108 @@
 # release notes
 ## current release
+### release-v17-20200908
+- release date: 2020-09-11
+- status: available
+- changes:
+  - pbta-histologies.tsv updates:
+    - clinical updates to columns: `aliquot_id`, `parent_aliquot_id`, `reported_gender`, `race`, `ethnicity`, `tumor_descriptor`, `primary_site`, `age_at_diagnosis_days`, `pathology_diagnosis`, `integrated_diagnosis`, `short_histology`, `broad_histology`, `Notes`, `OS_days`, `OS_status`, `age_last_update_days`,`seq_center`, `cancer_predispositions`, `molecular_subtype`
+    - Updated logic for `integrated_diagnosis`, `Notes`, `short_histology` and `broad_histology`: If the sample was subtyped, values were updated based on new subtyping, if the `pathology_diagnosis` changed between v16 and v17, all values were set to `NA` and will be updated in the next release, all remaining samples were set to the values from v16.
+    - changed column `glioma_brain_region` to `CNS_region`, updated logic and entities, and assigned regions for all tumors
+    - added columns `pathology_free_text_diagnosis` and `cohort_participant_id`
+    - added sequencing center for `BS_QB84TBA3` and `BS_S2TA8R29` (previously NA)
+    - aliquot_id for `BS_MVYA262V` sample_id `7316-3213` was changed from `A05265` to `A05233`
+    - updated medulloblastoma subtypes per [#742](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/742)
+    - added LGAT subtyping per [#631](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/631)
+    - added EWS diagnoses per [#623](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/623)
+  - new consensus mutation and TMB files from [TCGA analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) :
```

Comment:
```suggestion
  - update to the PBTA TMB file `pbta-snv-consensus-mutation-tmb-coding.tsv` post bug fix, etc. (#727 and #728)
  - new consensus mutation and TMB files from [TCGA analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) :
```

Reply:
going to commit this because I updated the release notes to include this file on s3 per [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/764#issuecomment-691309219)